src/main/kotlin/study/racing/domain/CarName.kt

```diff
@@ -0,0 +1,14 @@
+package study.racing.domain
+
+class CarName(
+    carName: String
+) {
+    var name = carName
+        private set
```

Comment:
```suggestion
class CarName(
    val carName: String
) {
```

변경 되지 않는 carName은 프로퍼티로 가도 괜찮을 것 같아요.

Reply:
불변인 객체 생성기준으로 수정하겠습니다!